pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdPersistentTopics.java

```diff
@@ -507,8 +512,23 @@ void run() throws PulsarAdminException {
             } else {
                 messageId = validateMessageIdString(messageIdStr);
             }
-
-            getPersistentTopics().createSubscription(persistentTopic, subscriptionName, messageId);
+            Map<String, String> map = new HashMap<>();
+            if (properties != null) {
+                for (String property : properties) {
+                    if (!property.contains("=")) {
```

Comment:
this condition is useless since if `=` is absent, the keyValue length will be 1

Reply:
this was copy/paste code, I have consolidated all duplicate code in one single method and addressed your comment